browserdebuggertools/eventhandlers.py

```diff
@@ -64,11 +64,9 @@ def get_current_url(self):
         self.check_page_load()
         return self._url
 
-    def get_page_source(self):
+    def get_root_node_id(self):
```

Comment:
Could be called `get_root_backend_node_id()` so it doesn't get confused for the `nodeId` over the backendNodeId` since it's no longer private.